builds/e2e/longhaul.yaml

```diff
@@ -0,0 +1,64 @@
+trigger: none
+
+jobs:
+
+################################################################################
+  - job: LinuxAMD64Moby
+################################################################################
+    displayName: Linux AMD64 Moby
+    pool:
+      name: Azure-IoT-Edge-Core
```

Comment:
I would check but I think we typically set the pool name via a build variable

Reply:
No.  I checked existing yamls; but use variable for windows pipeline in ci\dotnet.yaml only.  I will keep it simple for now.